package.json

```diff
@@ -2,8 +2,9 @@
     "name": "tabsoutliner", 
     "license": "MPL 2.0", 
     "author": "Bernard `Guyzmo` Pratz", 
-    "version": "0.4", 
+    "version": "0.0.4", 
```

Comment:
I changed it to 0.0.4 just because JPM kept throwing a warning about not using semver:

```
JPM [warning] The `version` entry must be specified, and it should follow semantic
versioning rules. For example, it must be a string containing three
values indicating major, minor, and patch versions. For example:

"1.4.56"
"0.0.1"
"1.0.0rc4"

http://semver.org
```

How about using 0.5.0?